Functions.Templates/Templates-v2/TimerTrigger-Python/template.json

```diff
@@ -89,7 +87,7 @@
             "name": "appendFileContent_FunctionApp",
             "type": "AppendToFile",
             "createIfNotExists" : false,
-            "source": "$(FUNCTION_APP_TRANSFORMED)",
+            "source": "$(TIMER_FUNCTION_BODY)",
```

Comment:
Thanks for catching this. `$(AppBody)` should also be changed to `$(TIMER_FUNCTION_BODY)` in `function_app.py`.

Reply:
So far, I have only tried it "appendToFile" actions therefore didn't come across the mentioned issue.